playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -585,6 +585,16 @@ EDXAPP_PRIVATE_REQUIREMENTS:
     # Caliper and xAPI event routing plugin
     - name: edx-event-routing-backends==5.5.6
 
+# List of additional npm packages that should be installed into the
+# edxapp virtual environment.
+# `name` (required), `version` (optional), and `extra_args` (optional)
```

Comment:
[question] Is this `extra_args` option used when installing these packages below?

Reply:
I agree that I didn't include them in the actual `shell` module, and I don't think it's necessary to support at the moment. I'll leave it off until we have a need for it.